src/modules/card/card.service.ts

```diff
@@ -119,87 +126,73 @@ export class CardService {
-        createCardDTO.userDefinitiveSolutionName = userDefinitiveSolution.name;
-      }
-
-      if (createCardDTO.userAppDefinitiveSolutionId) {
-        const userAppDefinitiveSolution = await this.userService.findById(
-          createCardDTO.userAppDefinitiveSolutionId,
-        );
-        if (!userAppDefinitiveSolution)
-          throw new NotFoundCustomException(NotFoundCustomExceptionType.USER);
-        createCardDTO.userAppDefinitiveSolutionName =
-          userAppDefinitiveSolution.name;
-      }
-
-      if (createCardDTO.managerId) {
-        const manager = await this.userService.findById(
-          createCardDTO.managerId,
-        );
-        if (!manager)
-          throw new NotFoundCustomException(NotFoundCustomExceptionType.USER);
-        createCardDTO.managerName = manager.name;
-      }
-
-      const lastInsertedCard = await this.cardRepository.find({order: {id: 'DESC'}, take: 1})
-      const {siteCardId} = lastInsertedCard[0]
-      createCardDTO.siteCardId = siteCardId+1
-      createCardDTO.siteCode = site.siteCode
-      createCardDTO.cardTypeColor = cardType.color
-      createCardDTO.areaName = area.name
-      createCardDTO.level = area.level
-      createCardDTO.superiorId = Number(area.superiorId) === 0 ? area.id: area.superiorId
-      createCardDTO.priorityCode = priority.priorityCode
-      createCardDTO.priorityDescription = priority.priorityDescription
-      createCardDTO.cardTypeMethodology = cardType.cardTypeMethodology
-      createCardDTO.cardTypeMethodologyName = cardType.methodology
-      createCardDTO.cardTypeName = cardType.name
-      createCardDTO.preclassifierCode = preclassifier.preclassifierCode
-      createCardDTO.preclassifierDescription = preclassifier.preclassifierDescription
-      createCardDTO.creatorName = creator.name
-      createCardDTO.responsableName = responsible.name
-      createCardDTO.createdAt = new Date()
-      createCardDTO.cardDueDate = new Date()
+      var lastInsertedCard = await this.cardRepository.find({
+        order: { id: 'DESC' },
+        take: 1,
+      });
+      const { siteCardId } = lastInsertedCard[0];
+
+      const card = await this.cardRepository.create({
+        ...createCardDTO,
+        siteCardId: siteCardId + 1,
+        siteCode: site.siteCode,
+        cardTypeColor: cardType.color,
+        areaName: area.name,
+        level: area.level,
+        superiorId: Number(area.superiorId) === 0 ? area.id : area.superiorId,
+        priorityCode: priority.priorityCode,
+        priorityDescription: priority.priorityDescription,
+        cardTypeMethodology: cardType.cardTypeMethodology,
+        cardTypeMethodologyName: cardType.methodology,
+        cardTypeName: cardType.name,
+        preclassifierCode: preclassifier.preclassifierCode,
+        preclassifierDescription: preclassifier.preclassifierDescription,
+        creatorName: creator.name,
+        responsableName: responsible.name,
```

Comment:
can se set this nullable? or is required?

Reply:
looks like responsable and responsable id we update it in another service, can se remove it? set null or EMPTY values

Reply:
Im not seeing the database right now but at least responsible id is not null

Reply:
Let me see if i can make it nullable

Reply:
I make them nullables, can u check it again?

Reply:
also de name is not null

Reply:
can u check it again the values?